.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: django-prbac test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    
+    strategy:
+      matrix:
+        python-version: [ "3.7", "3.8", "3.9" ]
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: install dependencies
+        run: |
+          pip install django==3.2.* 
```

Comment:
Might be worth adding another matrix dimension for Django versions and run against 3.2 and 4.2.

Reply:
See https://github.com/dimagi/django-prbac/pull/72